concent_api/core/tests/test_core_views.py

```diff
@@ -195,10 +196,7 @@ def test_send_should_return_http_400_if_data_is_incorrect(self):
             ),
             content_type                        = 'application/octet-stream',
         )
-        self._test_400_response(
-            response,
-            error_code=ErrorCode.MESSAGE_VALUE_BLANK
-        )
+        self._test_400_response(response)
```

Comment:
There is no error code in this response. Check `common.decorators.require_golem_message` - there are cases when it is not passed to response. I have added checking beginning of `error_message`.

>  think that error_message and error_code should be obligatory, not optional. 

`error_code` is not always available, and we decided a long time ago that we don't want to check `error_message` until it is necessary - because they might change too often (but really they don't).

Reply:
ok...It is quite strange, sometimes there are erorr_codes, sometimes not. But I agree with explanation.